app/apollo/test/channel.fga.spec.js

```diff
@@ -0,0 +1,666 @@
+/**
+ * Copyright 2020 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { expect } = require('chai');
+const fs = require('fs');
+const { v4: UUID } = require('uuid');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+
+const { models } = require('../models');
+const resourceFunc = require('./api');
```

Comment:
I suspect this was simply copied from previously existing test files, where it was also named `resourceFunc`.... but it's not a great name for the `./api` import.  For historical reasons, the `api.js` file mixed authentication GQL functions like `signUp` and `signIn` with `resource` GQL functions like `resourceHistory`.

In the case of _**this**_ code (and likely all the other new FGA test code), the import is for the "authentication API" functionality.  It's **only** used for user sign-in.  Use better names like `authFunc`/`authApi`;

Reply:
Changed in all edited fga.spec.js files

Reply:
Resolved